src/tokens.ts

```diff
@@ -266,6 +276,16 @@ export const TOKEN_SYMBOLS_MAP = {
     },
     coingeckoId: "tether",
   },
+  WAZERO: {
+    name: "Wrapped AZERO",
+    symbol: "WAZERO",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.ALEPH_ZERO]: "0xb7Da55D7040ef9C887e20374D76A88F93A59119E",
+      [CHAIN_IDs.MAINNET]: "0xdD0ae774F7E300CdAA4EA371cD55169665Ee6AFe",
```

Comment:
Mixing wrapped and non-wrapped addresses like this feels a bit like roller skating on ice. We do it for MATIC/WMATIC but it doesn't feel nice.